libres_util/include/ert/res_util/res_log.h

```diff
@@ -28,6 +28,16 @@ extern "C" {
 
 #include <ert/util/log.h>
 
+//Same as pythons default log levels.
+typedef enum {
+  CRITICAL=0, //OOM.
+  ERROR=1, //When something we really expected to work does not, e.g. IO failure.
+  WARNING=2, //Important, but not error. E.g. combination of settings which can be intended, but probably are not.
+  INFO=3, //Entering functions/parts of the code
+  DEBUG=4 //Inside the for-loop, when you need the nitty gritty details. Think TRACE.
+} message_level;
```

Comment:
Should be `message_level_enum` or `message_level_type` - the identifer `message_level` alone will look like variable.